specification/quota/resource-manager/Microsoft.Quota/stable/2021-03-15/examples/putMachineLearningServicesQuotaRequestDedicated.json

```diff
@@ -0,0 +1,55 @@
+{
+  "parameters": {
+    "resourceUrl": "management.azure.com/subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.MachineLearningServices/locations/eastus",
+    "resourceName": "StandardDv2Family",
+    "api-version": "2021-03-15",
+    "createQuotaRequest": {
+      "properties": {
+        "limit": 200,
+        "unit": "Count",
+        "name": {
+          "value": "StandardDv2Family"
+        },
+        "resourceType": "dedicated"
+      }
+    }
+  },
+  "responses": {
+    "201": {
+      "body": {
+        "id": "/subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.Quota/resourceProviders/Microsoft.MachineLearningServices/locations/eastus/serviceLimitsRequests/2B5C8515-37D8-4B6A-879B-CD641A2CF605",
+        "name": "2B5C8515-37D8-4B6A-879B-CD641A2CF605",
+        "type": "Microsoft.Quota/ServiceLimits",
+        "properties": {
+          "provisioningState": "InProgress",
+          "message": ""
+        }
+      }
+    },
+    "200": {
+      "body": {
+        "id": "/subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.Quota/resourceProviders/Microsoft.MachineLearningServices/locations/eastus/serviceLimitsRequests/2B5C8515-37D8-4B6A-879B-CD641A2CF605",
+        "name": "2B5C8515-37D8-4B6A-879B-CD641A2CF605",
+        "type": "Microsoft.Quota/serviceLimits",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "message": "Request succeeded",
+          "properties": {
+            "properties": {
```

Comment:
>properties [](http://example.com/codeflow?start=13&length=10)

Why do you have `properties` so many times in the response?

Reply:
The nested properties are required by the ARM template, else the ARM template doesn't work.
Again followed the example from Compute, as pointed by ARM team previously.

Reply:
"This is confusing, because there is a swagger concept of “properties”, as well as an ARM resource concept of “properties”. So to output a property named “properties”, you need two levels of nesting for properties. " - Previous comment from ARM team.

Reply:
On the swagger side yes, but these are examples and should be closer to what the JSON would look like over the wire (not entirely). Take a look at this: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/resources/resource-manager/Microsoft.Resources/stable/2021-05-01/examples/TemplateSpecsCreate.json.

Reply:
Btw, ARM templates do not require nesting multiple `properties` properties together. I'm confused by that statement.

Reply:
The generated SDK needs it to parse the requests from ARM template.

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/b422712397a4b4fcacdc607a75f799611af43f57/specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/compute.json#L7905 - Example from Compute using nested properties.

Reply:
That's what suggested by SDK and ARM team with compute example.